src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -0,0 +1,2583 @@
+  "gui.tconstruct.tinker_station.repair_limited": "修复&修改",
+  "gui.tconstruct.tinker_station.repair_limited.description": "允许创建所有基本工具并应用大多数修改器。从左侧的按钮中选择布局开始。",
+  "gui.tconstruct.tinker_station.repair": "修复&修改",
+  "gui.tconstruct.tinker_station.repair.description": "创建小型和广泛的工具并修改现有工具以实现您的梦想！从左侧的按钮中选择一个布局即可开始。",
+  "gui.tconstruct.tinker_station.modifiers": "修改",
+  "gui.tconstruct.tinker_station.upgrades": "升级&技能",
+  "gui.tconstruct.tinker_station.traits": "特征",
+  "gui.tconstruct.tinker_station.components": "组件",
+  "gui.tconstruct.tinker_station.info": "维修或更换工具的主要方法。在左侧放置一个工具和一个材料即可开始。",
+  "gui.tconstruct.tinker_station.no_traits": "无:[",
+
+  "gui.tconstruct.manual.cycle.recipes": "跳到下个材料",
+
+  "recipe.tconstruct.tool_repair.fully_repaired": "工具未损坏",
+  "recipe.tconstruct.damaging.broken": "工具已经坏了",
+  "recipe.tconstruct.overslime.at_capacity": "工具没有多余的空间容纳超粘液",
+  "recipe.tconstruct.remove_modifier.no_modifiers": "工具没有任何可移除的修饰符",
+
+  "recipe.tconstruct.part_swapping.too_many_parts": "表格太小,无法在此工具上交换零件",
+
+  "recipe.tconstruct.modifier.not_enough_slot": "需要至少 1 %s 槽才能添加此修饰符",
+  "recipe.tconstruct.modifier.not_enough_slots": "需要至少 %s %s 个插槽才能添加此修饰符",
+  "recipe.tconstruct.modifier.validate_slots": "此更改使工具的 %s 插槽太少",
+  "recipe.tconstruct.modifier.max_level": "%s 等级不能高于 %s",
+  "recipe.tconstruct.modifier.requirements_error": "其中一个修改器的工具状态无效。请联系您的模组包制造商并请求更具描述性的错误。",
+
+
+  "_comment": "GUIs Messages",
+
+  "gui.tconstruct.caption": "标题",
+  "gui.tconstruct.error": "错误",
+  "gui.tconstruct.warning": "警告",
+  "gui.tconstruct.general.hover": "将鼠标悬停在条目上以获取更多信息",
+  "gui.tconstruct.error.invalid_pattern": "模式不包含有效的工具部件！",
+  "gui.tconstruct.error.invalid_recipe": "无效配方！",
+  "gui.tconstruct.error.unknown_material": "未知材料",
+  "gui.tconstruct.error.useless_tool_part": "此部件不能用于制作任何工具！\n\n材料 %s 缺少某些信息,或者没有工具在其制作中使用 (n) %s。",
+  "gui.tconstruct.error.not_enough_durability": "没有足够的耐久度来更换零件！\n需要更多的耐久度。",
+
+
+  "_comment": "Subtitles for sounds",
+
+  "subtitles.tconstruct.little_saw": "使用中锯子",
+  "subtitles.tconstruct.item_frame_click": "物品框架点击",
+  "subtitles.tconstruct.casting_cools": "施放物品冷却",
+
+  "subtitles.tconstruct.slime_sling": "史莱姆弹跳",
+  "subtitles.tconstruct.slime_sling.teleport": "史莱姆弹弓传送",
+  "subtitles.tconstruct.enderporting": "末影传送",
+  "subtitles.tconstruct.throw.throwball": "投掷球",
+  "subtitles.tconstruct.throw.shuriken": "手里剑",
+
+  "subtitles.tconstruct.necrotic_heal": "生命枯竭",
+  "subtitles.tconstruct.extra_jump": "实体猛冲",
+
+  "subtitles.tconstruct.slime_teleport": "史莱姆传送",
+  "subtitles.tconstruct.slimy_bounce": "史莱姆弹跳",
+
+  "subtitles.tconstruct.equip.slime": "史莱姆盔甲晃动",
+  "subtitles.tconstruct.equip.travelers": "旅行者盔甲沙沙作响",
+  "subtitles.tconstruct.equip.plate": "板甲铮铮",
+
+  "subtitles.tconstruct.block.earth_crystal.chime": "大地水晶钟声",
+  "subtitles.tconstruct.block.sky_crystal.chime": "天空水晶钟声",
+  "subtitles.tconstruct.block.ichor_crystal.chime": "灵液水晶钟声",
+  "subtitles.tconstruct.block.ender_crystal.chime": "末影水晶钟声",
```

It should be better to make translation of "chime" match vanilla Minecraft.
将“chime”的译文与原版Minecraft保持一致会更好。

```suggestion
  "subtitles.tconstruct.block.earth_crystal.chime": "大地水晶：叮铃",
  "subtitles.tconstruct.block.sky_crystal.chime": "天空水晶：叮铃",
  "subtitles.tconstruct.block.ichor_crystal.chime": "灵液水晶：叮铃",
  "subtitles.tconstruct.block.ender_crystal.chime": "末影水晶：叮铃",
```

[chime >> 叮铃](https://minecraft.fandom.com/zh/wiki/%E7%B4%AB%E6%B0%B4%E6%99%B6%E5%9D%97#.E9.9F.B3.E6.95.88)